### PR TITLE
PAINTROID-280 Refactor JpgInfoDialog, PngInfoDialog, OraInfoDialog to Kotlin

### PR DESCRIPTION
**Pull request by CodeChamp-SS**
Files that were refactored:

1. JpgInfoDialog
2. PngInfoDialog
3. OraInfoDialog

https://jira.catrob.at/browse/PAINTROID-280

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by FromG — APPROVED** [review]
ok.

File: Paintroid/src/main/java/org/catrobat/paintroid/dialog/JpgInfoDialog.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        AlertDialog.Builder(requireContext(), R.style.PocketPaintAlertDialog)
+            .setMessage(R.string.pocketpaint_jpg_message_dialog)
+            .setTitle(R.string.pocketpaint_jpg_title_dialog)
+            .setPositiveButton(R.string.pocketpaint_ok) { dialog, which -> dismiss() }
+            .create()
+
+    companion object {
+        @JvmStatic
+        fun newInstance() = JpgInfoDialog()
+    }
```
Comment (gPathpp) [inline Paintroid/src/main/java/org/catrobat/paintroid/dialog/JpgInfoDialog.kt:38]:
```suggestion
```
That's an old pattern from the deprecated DialogFragment.
Please replace the new instance with the constructor call.

Comment (gPathpp) [reply Paintroid/src/main/java/org/catrobat/paintroid/dialog/JpgInfoDialog.kt:38]: Please do it for the other classes too. Thx!

**Review by gPathpp — CHANGES_REQUESTED** [review]

File: Paintroid/src/main/java/org/catrobat/paintroid/dialog/JpgInfoDialog.kt (hunk 1 above)
Comment (CodeChamp-SS) [reply Paintroid/src/main/java/org/catrobat/paintroid/dialog/JpgInfoDialog.kt]: done 🙂

File: Paintroid/src/main/java/org/catrobat/paintroid/dialog/JpgInfoDialog.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ */
+package org.catrobat.paintroid.dialog
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatDialogFragment
+import org.catrobat.paintroid.R
+
+class JpgInfoDialog : AppCompatDialogFragment() {
```
Comment (gPathpp) [inline Paintroid/src/main/java/org/catrobat/paintroid/dialog/JpgInfoDialog.kt:27]: Nice :+1:

**Review by gPathpp — APPROVED** [review]
LGTM :+1:

**Pull request merged**